In [4]:
import numpy as np
import pandas as pd

In [5]:
df = pd.read_csv("df_final_unnormalized.csv")

### Cattle Interpretations

In [6]:
range = df["cattle"].max() - df["cattle"].min()
min = df["cattle"].min()

range

42324918.67

In [7]:
fraction = 1000000/range

In [8]:
np.exp(fraction*0.219886)

1.0052087085367982

for an increase in 1 million cows, water stress increases by a factor of 1.0052

### Hydroelectric Interpretations

In [11]:
range = df["hydro_electricity"].max() - df["hydro_electricity"].min()
min = df["hydro_electricity"].min()

range

99.97862299179452

In [12]:
fraction = 1 / range

In [13]:
np.exp(fraction*-2.146425)

0.9787599757464195

for an increase in hydro electricity of 1%, water stress decreases by a factor of 0.987